IdentityServer/v5/AspNetIdentity/IdentityServerAspNetIdentity/IdentityServerAspNetIdentity.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Duende.IdentityServer.AspNetIdentity" Version="5.1.0" />
```

Comment:
Are we still maintaining the v5 samples, especially given that v5 is out of support?

Reply:
I think it's ok to bump the version numbers, but no plans to maintain the v5 versions beyond that.